pkg/scheduler/schedule_one_test.go

```diff
@@ -128,6 +129,9 @@ func (f *fakeExtender) Prioritize(
 func (f *fakeExtender) Bind(binding *v1.Binding) error {
 	if f.isBinder {
 		f.gotBind = true
```

Comment:
if bind error, I think f.gotBind should be false. WDYT？

Reply:
Right, If extender bind error, should not give "gotBind" flag